api/predict.py

```diff
@@ -44,7 +44,11 @@ def post(self):
         text = self.model_wrapper.read_text(text_data)
         preds = self.model_wrapper.predict(text)
 
+        if preds is None:
+            result['status'] = 'error'
+        else:
+            result['status'] = 'ok'
+
         result['pred_txt'] = preds
```

Comment:
Should we have `result['pred_txt'] = None` if `preds is None`? Otherwise this PR LGTM.

Reply:
Since `result['pred_txt']` is being set to `preds` if `preds` is none then `result['pred_txt']` will also be none

Reply:
Sorry for the confusion, I meant the other way: Do we intend that  `result['pred_txt'] = None` if `preds is None`?